### ISSUE TITLE
I hope you can add subcategories,Or can filter the category description in the search bar.

### ISSUE THREAD

**Issue by woshilhz001** [sunnyark/civitai-shortcut#43]
I hope you can add subcategories,Or can filter the category description in the search bar.

**Comment by sunnyark** [comment]
@woshilhz001 Do you want to categorize one step further than the existing classification? We haven't been considering beyond the current state when dividing the classification items. I will think about it for a moment.

**Comment by AugmentedRealityCat** [comment]
I am a new user of this extension and I am trying to find how to search exclusively for things made for model 1.5, or model 2.1. This is something that is very useful on the Civitai website itself. 

What is the simplest way to, let's say, get a list of all my LoRAs and Lycoris that are compatible with model 2.1 ? Thanks in advance - I hope it's OK this ask this question here in this thread.

**Comment by sunnyark** [comment]
@AugmentedRealityCat I will soon be adding a feature to filter using the basemodel provided by Civitai. Thank you for your suggestion.

**Comment by AugmentedRealityCat** [comment]
> edit: I updated it a little while ago. If you update the extension, it will be reflected.

YES ! It works very well. Very nice that you can select multiple categories. It's exactly what I wanted.
Thank you so much @sunnyark

**Comment by woshilhz001** [comment]
@sunnyark 
Thanks, my categorization may be confusing

-tools
    -colors
    -cloths
    -details
    -......
-styles 
    -Asia 
    -Europe 
    -2D
    -3D
    -......
-......

I wish I could search @ tools as well as @cloths.
I don't know much about programming, would it be better if I could add keyword search to the categories?

**Comment by sunnyark** [comment]
@woshilhz001 When we first designed the categorization, we couldn't create it to include the ability to have subcategories.
To add this feature, we will need to redesign it from scratch. of course, some users might want 2-3 levels or even more :), but we will consider reasonable ways to address this.

**Comment by sunnyark** [comment]
@woshilhz001 Suddenly, I came up with a great idea, I implemented it, and the results weren't bad. I'd like to hear your opinion. Increasing the classification items in multiple levels didn't seem like a good solution in terms of UI or UX. So, one shortcut to multiple categories For example, if you put one shortcut in the 'Tools' category and also register it in the 'Colors' category, it will appear when you select 'Tools' and 'Colors' in the Search category section. It's hard to explain in words, but I'll update when testing is complete.

**Comment by woshilhz001** [comment]
Thank you for what you do!
I would like to add keywords to the description of the category.
For example:
colors: [tools],[colors],[...],description
cloths: [tools],[cloths],[...],description
details: [tools],[details],[...],description
This way typing @#toos or @ #colors in the search bar will filter out models containing the keywords.
This may not be as easy as I thought, maybe your shortcut method would be better
Sorry if I'm wrong.

**Comment by sunnyark** [comment]
@woshilhz001 Hmm, it seems you're trying to substitute keywords for classification levels. However, that approach might pose a problem as users would need to remember the keywords. I'll think about it more, but for now, I recommend trying out the classification method you updated this time. 

eg)
If you have a shortcut that you want to belong to the 'tools / colors' categories. , you create classifications for 'tools' and 'colors' and then register the shortcut under both categories. When using it, you can narrow down the scope by selecting both 'tools' and 'colors' from the classification options in the search section.
